FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +22,60 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let _ = self else {
+				return
+			}
```

Comment:
`self` is not used inside this block, thus there's no need to unwrap it. Simple check if `self` is not `nil` is enough.

Please replace unwrapping with nullability check

```suggestion
			guard self != nil else {
				return
			}
```

Reply:
It makes sense to me. Updated.